rust/lance-encoding/src/compression.rs

```diff
@@ -781,6 +822,52 @@ impl DecompressionStrategy for DefaultDecompressionStrategy {
                     general.compression.as_ref().expect_ok()?.scheme(),
                 )?))
             }
+            Compression::VariablePackedStruct(description) => {
```

Comment:
Why do we need a block compressor?

Reply:
Sorry, I didn’t understand this comment. What does “block compressor” refer to here?